diff --git a/pcs/pcs.c b/pcs/pcs.c
index 750dc97..7dadc20 100644
--- a/pcs/pcs.c
+++ b/pcs/pcs.c
@@ -14,7 +14,7 @@
 #include "pcs_mem.h"
 #include "pcs_utils.h"
 #include "pcs_http.h"
-#include "cJSON.h"
+#include "cjson/cJSON.h"
 #include "pcs.h"
 #include "pcs_passport_dv.h"
 
diff --git a/shell.c b/shell.c
index 0641b8a..cec83f7 100644
--- a/shell.c
+++ b/shell.c
@@ -26,7 +26,7 @@
 #endif
 
 #include "pcs/pcs_mem.h"
-#include "pcs/cJSON.h"
+#include "cjson/cJSON.h"
 #include "pcs/pcs_utils.h"
 #include "pcs/pcs.h"
 #include "rb_tree/red_black_tree.h"
